packages/x-components/src/x-modules/search/store/types.ts

```diff
@@ -73,14 +74,16 @@ export interface SearchGetters {
   /** The adapter request object for retrieving the results, or null if there is not
    * valid data to create a request. */
   request: InternalSearchRequest | null;
+  /** The combination of the query and the selected related tags. */
+  query: string;
```

Comment:
I decided to leave this `query` and the `setQuery` mutation in every module so we can link it through the documentation.

Reply:
I was finishing the review and I just saw this, you are so good that answer my questions before I asked them 🤣